tap_github/schemas/comments.json

```diff
@@ -93,6 +99,199 @@
+              "type": ["null", "string"]
+            },
+            "html_url": {
+              "type": ["null", "string"]
+            },
+            "followers_url": {
+              "type": ["null", "string"]
+            },
+            "following_url": {
+              "type": ["null", "string"]
+            },
+            "gists_url": {
+              "type": ["null", "string"]
+            },
+            "starred_url": {
+              "type": ["null", "string"]
+            },
+            "subscriptions_url": {
+              "type": ["null", "string"]
+            },
+            "organizations_url": {
+              "type": ["null", "string"]
+            },
+            "repos_url": {
+              "type": ["null", "string"]
+            },
+            "events_url": {
+              "type": ["null", "string"]
+            },
+            "received_events_url": {
+              "type": ["null", "string"]
+            },
+            "type": {
+              "type": ["null", "string"]
+            },
+            "site_admin": {
+              "type": ["null", "boolean"]
+            },
+            "starred_at": {
+              "type": ["null", "string"]
+            }
+          }
+        },
+        "name": {
+          "type": ["null", "string"]
+        },
+        "description": {
+          "type": [
+            "null",
+            "string"
+          ]
+        },
+        "external_url": {
+          "type": ["null", "string"]
+        },
+        "html_url": {
+          "type": ["null", "string"]
+        },
+        "created_at": {
+          "type": ["null", "string"],
+          "format": "date-time"
+        },
+        "updated_at": {
+          "type": ["null", "string"],
+          "format": "date-time"
+        },
+        "permissions": {
+          "type": ["null", "object"],
+          "properties": {
+            "issues": {
+              "type": ["null", "string"]
+            },
+            "checks": {
+              "type": ["null", "string"]
+            },
+            "metadata": {
+              "type": ["null", "string"]
+            },
+            "contents": {
+              "type": ["null", "string"]
+            },
+            "deployments": {
+              "type": ["null", "string"]
+            }
+          },
+          "additionalProperties": {
+            "type": ["null", "string"]
+          }
+        },
+        "events": {
+          "type": ["null", "array"],
+          "items": {
+            "type": ["null", "string"]
+          }
+        },
+        "installations_count": {
+          "type": ["null", "integer"]
+        },
+        "client_id": {
+          "type": ["null", "string"]
+        },
+        "client_secret": {
+          "type": ["null", "string"]
+        },
+        "webhook_secret": {
+          "type": ["null", "string"]
+        },
+        "pem": {
+          "type": ["null", "string"]
+        }
+      }
+    },
+    "reactions": {
```

Comment:
Used this at many places, move this into common.

Reply:
Added a shard schema for the same